config/config_test.go

```diff
@@ -276,16 +276,17 @@ func TestEmptyFieldsAndRegex(t *testing.T) {
 	var expectedConf = Config{
 
 		Global: &GlobalConfig{
-			ResolveTimeout:   model.Duration(5 * time.Minute),
-			SMTPSmarthost:    "localhost:25",
-			SMTPFrom:         "alertmanager@example.org",
-			HipchatAuthToken: "mysecret",
-			HipchatURL:       "https://hipchat.foobar.org/",
-			SlackAPIURL:      "mysecret",
-			SMTPRequireTLS:   true,
-			PagerdutyURL:     "https://events.pagerduty.com/generic/2010-04-15/create_event.json",
-			OpsGenieAPIHost:  "https://api.opsgenie.com/",
-			VictorOpsAPIURL:  "https://alert.victorops.com/integrations/generic/20131114/alert/",
+			ResolveTimeout:    model.Duration(5 * time.Minute),
+			SMTPSmarthost:     "localhost:25",
+			SMTPFrom:          "alertmanager@example.org",
+			HipchatAuthToken:  "mysecret",
+			HipchatURL:        "https://hipchat.foobar.org/",
+			SlackAPIURL:       "mysecret",
+			SMTPRequireTLS:    true,
+			PagerdutyURL:      "https://events.pagerduty.com/v2/enqueue",
+			PagerdutySeverity: "error",
```

Comment:
We don't have similar settings as global config options for other notifiers.

Reply:
removing this option from the global config